packages/docs/docs/api/configuration/eslint-plugin.mdx

```diff
@@ -16,7 +16,7 @@ sidebar_position: 2
 type Options = {
   // Possible strings where you can import stylex from
   //
-  // Default: ['@stylexjs/stylex']
+  // Default: ['stylex', '@stylexjs/stylex']
```

Comment:
Existing code already had `stylex` in validImports by default so I aligned the default for all rules and reflected it in the docs too.